app/assets/stylesheets/publishing.scss

```diff
@@ -98,3 +98,21 @@ ul.decided_item li.type_and_title {
   font-weight: bolder;
   margin-top: 20px;
 }
+
+.waiting_approval_items {
+  padding: 0.5em 15px;
+  margin: 0.3em 0em;
+}
+
+.rejected_items {
+  padding: 0.5em 15px;
+  margin: 0.3em 0em;
+}
+
+.red_boxed{
+  border: 1px solid rgba(200, 50, 100, .3);
+  border-radius: 3px;
+  background: rgba(200, 50, 100, .1);
+  padding: 0.5em 15px;
+  margin: 0.3em 0em;
+}
```

Comment:
I can do, but it has the unintended consequence of making the text red... which I found a bit aggressive... Here's a sample of what it is:

![Screenshot from 2023-06-13 16-32-58](https://github.com/seek4science/seek/assets/36694679/6eed1386-8eb4-49e5-bff6-f50fbbbe0bd6)

and what it would be:

![Screenshot from 2023-06-13 16-32-23](https://github.com/seek4science/seek/assets/36694679/f36962e8-9ff1-4877-8b80-c7df68f6ce62) 

I thought the current state is better, but if you want me to change it just confirm and I'll do it.

Reply:
Could not bring myself to have everything in red, so I removed the red box in the whole list, and instead turned the rejected items info to alert-danger:
![Screenshot from 2023-06-14 13-22-30](https://github.com/seek4science/seek/assets/36694679/eb3108b2-b671-4500-996c-3e954a73d8e1)

Resolved in [d6dd211](https://github.com/seek4science/seek/pull/1484/commits/d6dd211d8d88421755f33596273bf1193fa81099)